addon/ActivityStreams.js

```diff
@@ -190,9 +190,9 @@ ActivityStreams.prototype = {
    *                    TODO: Refactor this in to a different functions that handle refreshing data separately
    */
   _refreshAppState() {
-    console.log("Refresh app state");
```

Comment:
Just a thought. It would be nice to have some logging in place when running debug mode.